app/src/main/java/io/horizontalsystems/bankwallet/modules/market/search/CoinDataItemsAdapter.kt

```diff
@@ -37,13 +37,20 @@ class CoinDataItemsAdapter(private val onItemClick: (CoinDataViewItem) -> Unit)
 
             title.text = item.name
             subtitle.text = item.code
+
+            if(item.type is CoinType.Erc20 || item.type is CoinType.Bep20  || item.type is CoinType.Bep2)
+                rank.text = item.type::class.simpleName
+            else
+                rank.isVisible = false
```

Comment:
rank visibility should be set in the if clause too. otherwise we may face the issue when recycler view adapter reuses this viewholder